### ISSUE TITLE
Change Alt Text for Give Icon on Credit page

### ISSUE THREAD

**Issue by qiqicodes** [hackforla/website#1571]
### Prerequisite
Please make sure you have read the **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue

### Overview
As a developer, I want to provide updated alt text for the  `Give` icon so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [ ] Change the `alt` property value to 'Give icon' within `_data/internal/credits/give.yml`

**From**
`alt: 'Image of Give'`
**To:**
`alt: 'Give icon'`

#### File and Code links you will need to review this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/give.yml`

- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/90a20d1e088b0a05a286af101f82135ad4774e39/_data/internal/credits/give.yml#L10

**Comment by Aveline-art** [comment]
Unassigned due to inactivity. @gr4ssman Feel free to jump back in on an issue once you are ready to contribute!

**Comment by macho-catt** [comment]
hi @amnak1284, can you move this to the "in progress" column of the project board please? Thanks!

**Comment by ghost** [comment]
Progress: Currently I cloned the repository and was able to set up the project locally and get the site to run. The only thing left to do is change the alt text and then create the pull request
Blockers: Currently there are no blockers
Availability: I will have around 3 hours
ETA:  this issue should be done by tomorrow 
Pictures: no images of changes so far
